mmengine/runner/log_processor.py

```diff
@@ -263,6 +311,8 @@ def get_log_after_epoch(self,
         # message.
         log_items = []
         for name, val in chain(tag.items(), non_scalar_tag.items()):
+            if name in ('time', 'data_time'):
+                name = self._remove_prefix(name, f'{mode}/')
```

Comment:
If name in `('time', 'data_time')`, there is no need to remove the prefix of name.